apps/src/aichat/views/ChatEventsList.tsx

```diff
@@ -61,7 +61,7 @@ const WaitingAnimation: React.FunctionComponent<{shouldDisplay: boolean}> = ({
     return (
       <img
         src="/blockly/media/aichat/typing-animation.gif"
-        alt={'Waiting for response'}
+        alt={'Waiting for response'} // do we translate alt text?
```

Comment:
Bunch of comments we left sprinkled throughout these files that we'll need to resolve before merging

Reply:
to answer this one specifically, searched the codebase for 'alt=' and found a mix.  Seems like the right thing is to translate

Reply:
Do we need alt text here?  If it's decorative and the same information can be gleaned from other text nearby, then I think it's okay to have it as a blank string.

Reply:
There's no other text. Just this gif as in place of the next chat message https://github.com/code-dot-org/code-dot-org/blob/staging/apps/static/aichat/typing-animation.gif

Reply:
I guess to pull the thread a little more... is there a user scenario where they'll be able to make use of this alt text?  It sounds like it might show very quickly?

Reply:
I'm not sure how quick-draw folks with screen readers are-  average latency for a response is somewhere around 3-4 seconds if I'm reading the charts right. I think the use case would be more for those requests that take longer, a screen-reader user might navigate there to see what's taking so long.

Reply:
Fair enough!